src/components/Movies/LandPage/LandPage.styled.tsx

```diff
@@ -0,0 +1,130 @@
+import styled from "styled-components";
+import { ButtonStyle } from "../../Button/Button.styled";
+type props = {
+  bg: string;
+};
+export const LandPageStyle = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+`;
+
+export const HeaderStyle = styled.header`
+  h1 {
+    font-family: "Roboto Mono";
+    font-size: 48px;
+    font-weight: 500;
+  }
+`;
+export const SearchBoxStyle = styled.div`
+  position: relative;
+  input {
+    width: 300px;
+    height: 43px;
+    text-align: left;
+    padding-left: 16px;
+
+    &:focus::placeholder {
+      color: white;
+    }
+  }
+`;
+
+export const SearchIconButtonStyle = styled.button`
+  all: unset;
+  height: 50px;
+  position: absolute;
+  top: 0px;
+  right: 0px;
+  padding-left: 16px;
+  padding-right: 16px;
+  cursor: pointer;
+`;
+
+export const MoviesContainerStyle = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: 60px;
+  margin-bottom: 60px;
+  width: 1440px;
+  height: max-content;
+  row-gap: 48px;
+  column-gap: 75px;
+  justify-content: center;
+  align-items: center;
+
+  @media only screen and (max-width: 600px) {
```

Comment:
The media query for `max-width: 600px` and `max-width: 768px` are not necessary because they are covered by the media query `max-width: 1024px`.